docs/building-apps/ingest-sdk/ingestion-pipeline-code.mdx

```diff
@@ -0,0 +1,290 @@
+---
+title: Ingestion Pipeline Sample Code
+sidebar_position: 30
+---
+
+Complete code for a small sample of a Stellar network ingestion pipeline using the Stellar Go [Ingestion Sdk](overview.mdx#the-ingestion-sdk-packages) to publish derived data to a remote message broker. Demonstrate event driven, distributed processing with a sample microservice(python script) as subscriber.
+
+This example uses ZeroMQ [goczmq](https://github.com/zeromq/goczmq) go wrapper SDK, which requires a few o/s [dependent libraries be installed on host machine](https://github.com/zeromq/goczmq?tab=readme-ov-file#dependencies) also.
+
+Put these files in a directory, compile and run with `go build -o pipeline ./.; ./pipeline`
+
+### `go.mod`
+
+<CodeExample>
+
+```
+module github.com/stellar/example-ingestion-pipeline
```

Comment:
Should this be `module example/pipeline` based on [this](https://github.com/stellar/stellar-docs/pull/687/files#diff-633df6eef2d916e31d810cde32b690f03f48f6fad8da21af6831442a1f73a56bR25)?

Reply:
yes, good catch. https://github.com/stellar/stellar-docs/pull/687/commits/25dba75a7ddb4190eaba44a6916a471b1b08cbce